manifests/kustomize/base/metadata/metadata-envoy-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: container
-        image: gcr.io/ml-pipeline/envoy:initial
+        image: gcr.io/ml-pipeline/envoy:metadata-grpc
```

Comment:
I thought you mean MKP path failed.
Wondering whether we change it like MKP path to use 0.4.0 version like other containers?

Reply:
Haven't tried MKP path, from release.cloudbuild.yaml looks like it builds this image each time.

It would be good if we change like that, but I don't forsee any updates. I think this fix is fine for now.